src/visual.ts

```diff
@@ -246,6 +246,14 @@ module powerbi.extensibility.visual {
         private localizationManager: ILocalizationManager;
         private horizontalAutoScrollingPositionOffset: number = 200;
 
+        private selectedGranulaPos: number = null;
+
+        private scrollAutoFocusFunc = (selectedGranulaPos: number) => {
+            if (selectedGranulaPos) {
```

Comment:
Should we ignore 0 along with null and undefined?

Reply:
I suppose yes because 0 is default scroll position and there is no sense to apply this functionality in that case